pkg/core/transaction/witness_scope.go

```diff
@@ -53,3 +54,41 @@ func ScopesFromString(s string) (WitnessScope, error) {
 	}
 	return result, nil
 }
+
+// scopesToString converts witness scope to it's string representation. It uses
+// `, ` to separate scope names.
```

Comment:
So where this space comes from?

Reply:
It's from Flags serialisation in C#.
I wrote a tiny example:
```
using System;

namespace test
{
    [Flags]
    public enum MyFlag : byte
    {
        Green = 0x01,
        Read = 0x01,
        Blue = 0x10
    }

    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine(MyFlag.Green | MyFlag.Blue);
        }
    }
}
```

The output is `Green, Blue` (with space).

Reply:
Also wrote C# test to check:
```
        [TestMethod]
        public void Json_SeveralScopes()
        {
            var attr = new Cosigner()
            {
                Scopes = WitnessScope.CalledByEntry | WitnessScope.CustomContracts,
                Account = UInt160.Zero,
                AllowedContracts = new[] { UInt160.Zero },
            };

            var json = "{\"account\":\"0x0000000000000000000000000000000000000000\",\"scopes\":\"CalledByEntry, CustomContracts\",\"allowedContracts\":[\"0x0000000000000000000000000000000000000000\"]}";
            attr.ToJson().ToString().Should().Be(json);

            var copy = Cosigner.FromJson(JObject.Parse(json));

            Assert.AreEqual(attr.Scopes, copy.Scopes);
            Assert.AreEqual(attr.Account, copy.Account);
        }
```
So there should be a space.

Reply:
What if I'm to do
```
            var json = "{\"account\":\"0x0000000000000000000000000000000000000000\",\"scopes\":\"CalledByEntry,CustomContracts\",\"allowedContracts\":[\"0x0000000000000000000000000000000000000000\"]}";
```
and make a `FromJSON` for it?

Reply:
Well, technically it's OK not to have space between scopes, because Enum.Parse(...) is able to handle this. Should we serialise it without space?

Reply:
No, we're perfectly fine with generating spaces and accepting both spaced and unspaced version, same as with .net implementation.